dot/network/host.go

```diff
@@ -251,16 +241,18 @@ func (h *host) connect(p peer.AddrInfo) (err error) {
 // bootstrap connects the host to the configured bootnodes
 func (h *host) bootstrap() {
 	failed := 0
-	all := append(h.bootnodes, h.persistentPeers...)
-	for _, addrInfo := range all {
+	var allNodes []peer.AddrInfo
+	allNodes = append(allNodes, h.bootnodes...)
+	allNodes = append(allNodes, h.persistentPeers...)
+	for _, addrInfo := range allNodes {
```

Comment:
did deepsource complain about this? just curious what the issue was

Reply:
Yes https://deepsource.io/gh/ChainSafe/gossamer/run/7db3c103-c088-4fb5-904e-bad5ef591ab6/go/CRT-D0001
```
possibly assigns to a wrong variable
```